modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -1,126 +1,139 @@
-= Create and Configure an Outbound Message Flow
+= Create Outbound Message Flows
 
-You can create and configure an outbound message flow to receive application messages from your backend systems, validate and transform these messages, and then send the translated EDI message to your trading partners either directly, or through 3rd party connections.
+Create outbound message flows to receive application messages from your backend systems, validate and transform these messages to the format your partners expect, and then send them to your partners either directly, or through 3rd party connections.
 
-To send messages to a trading partner through a 3rd party connection, select an endpoint owned by that connection.
+Before you create your first message flow, configure your xref:setup-payload-storage-API.adoc[payload storage API].
```

Comment:
This is conveyed by way of a Before You Begin topic in the inbound message flow task, isn't it?
Either is fine; I kind of prefer this treatment when the prereq is just one thing....